source/hardware/controllers/numark_dj2go2_touch.rst

```diff
@@ -65,7 +65,7 @@ The pad mode is changed using the :hwlabel:`PAD MODE` button.
 ===============================================================  ======================================================================================================================
 Pad Mode                                                         Function
 ===============================================================  ======================================================================================================================
-:hwlabel:`CUES`                                                  Set cue points, and trigger them by clicking.
+:hwlabel:`CUES`                                                  Set cue points, and trigger them by clicking. Clear cue points by clicking with holding the :hwlabel:`PAD MODE` button.
```

Comment:
```suggestion
:hwlabel:`CUES`                                                  Set cue points, and trigger them by clicking. Clear cue points by clicking while holding the :hwlabel:`PAD MODE` button.
```

Reply:
Thank you.
I change from "with" to "while" and reset old commit .